db/seeds.rb

```diff
@@ -159,7 +159,7 @@ def create_default_lara_report
   auth_client = Client.where(name: "DEFAULT_REPORT_SERVICE_CLIENT").first_or_create(
     app_id: "DEFAULT_REPORT_SERVICE_CLIENT",
     app_secret: SecureRandom.uuid(),
-    domain_matchers: ".*\.concord\.org",
+    domain_matchers: ".*\.concord\.org localhost.*",
```

Comment:
Out of curiosity, does this raise any security concerns, given that anybody can run localhost?

Reply:
On a production server this localhost entry should be removed. 

A good TODO would be to add a note to the readme. 
This file is used to initialize a new database after which time it isn't used again. So an admin of a production server should go in and remove the localhost entry.

Reply:
I added a note to our Portal deployment google doc about this: https://docs.google.com/document/d/1dmAV4ojzwau2C-TANvoxw9jAnUN2F5FdOSSy6f42H84/edit#